src/txHashBuilder.c

```diff
@@ -452,6 +460,26 @@ void txHashBuilder_enterCertificates(tx_hash_builder_t* builder)
 	builder->state = TX_HASH_BUILDER_IN_CERTIFICATES;
 }
 
+static uint32_t getStakeCredentialSource(const stake_credential_type_t stakeCredentialType)
+{
+	enum {
+		KEY = 0,
+		SCRIPT = 1
+	};
+	switch (stakeCredentialType) {
+	case STAKE_CREDENTIAL_KEY_PATH:
+	case STAKE_CREDENTIAL_KEY_HASH:
+		return KEY;
+		break;
```

Comment:
`break`s are unnecessary here. I'd consider returning -1 in `default` after the assert (don't we get a warning for a missing return statement?).

Reply:
We use a _lot_ of unnecessary breaks in switches. Is there some rule for this we should adhere to?

Reply:
Not really a rule. If there is an obvious return (i.e. not hidden in a macro like in security policies), there is no need for `break`; otherwise I feel safer if break is used.